os/fs/driver/mtd/ftl.c

```diff
@@ -446,13 +446,13 @@ static int ftl_ioctl(FAR struct inode *inode, int cmd, unsigned long arg)
 	 * that command is just passed on to the MTD driver in a slightly
 	 * different form).
 	 */
-
+	printf("\n\n checkpoint %d\n\n", __LINE__);
```

Comment:
why are you using `printf` instead of `dbg`?

Reply:
Since this is a temp PR, this will be closed. Sorry for confusion.